src/object/authenticate.c

```diff
@@ -8656,20 +8656,20 @@ au_get_dba_user (void)
 }
 
 /*
- * au_check_serial_authorization - check whether the current user is able to
- *                                 modify serial object or not
+ * au_check_row_authorization - check whether the current user is able to
+ *                                 modify row object or not
  *   return: NO_ERROR if available, otherwise error code
- *   serial_object(in): serial object pointer
+ *   row_object(in): row object pointer
  */
-int
-au_check_serial_authorization (MOP serial_object)
+static int
+au_check_row_authorization (MOP row_object)
 {
   DB_VALUE creator_val;
   MOP creator;
   DB_SET *groups;
   int ret_val;
 
-  ret_val = db_get (serial_object, "owner", &creator_val);
+  ret_val = db_get (row_object, "owner", &creator_val);
 
   if (ret_val != NO_ERROR || DB_IS_NULL (&creator_val))
     {
```

Comment:
`ret_val = ER_QPROC_CANNOT_UPDATE_SERIAL;`
It should be modified. I think separating functions would be nice.

Reply:
Thanks for the thorough check.
Let's fix the error code to be distinct.